### DISCUSSION TITLE
Add option to show fallback colors on the sliders

### DISCUSSION THREAD

**Discussion by llakala** [evilmartians/oklch-picker#160] [General]
Would be nice to see what the fallback color *would* be for a given value

**Comment by ai** [comment]
If you select `hex` output, it will be always fallback

**Reply by llakala** [reply]
Really? Not what I'm seeing: 
[image](https://github.com/user-attachments/assets/cabf9ad0-8a6f-45cd-9a5a-be2d200c6e11)

**Reply by ai** [reply]
What do you mean by `fallback color` in this case?

I mean, that `fallback color` is the closest sRGB (old monitors) colors when you choose P3 color (new monitors).

 Something like here: https://oklch.com/#74.41,0.192,54,100

![Captura desde 2024-09-30 23-40-20](https://github.com/user-attachments/assets/d8286bf9-9376-45c8-8aa2-711b168ae6ea)

In your case, you choose sRGB safe color, so you don’t need fallback.

**Reply by llakala** [reply]
Oh, I meant that the sliders would display the fallback colors for a given value. Currently, the fallback is limited to when you have the color *selected*. I was saying it'd be useful to be able to see what the fallback color would be on the slider.

**Reply by ai** [reply]
You mean P3 color on top and sRGB fallback on the bottom?

**Comment by ai** [comment]
Done 24edca2

![photo_2024-10-04_01-47-25](https://github.com/user-attachments/assets/2120a3c0-7b06-44c3-b81a-978bb796ef22)

sRGB fallback above and P3 below (it is better to check in real app, since image has P3 → sRGB reduction).
